### ISSUE TITLE
Implement import from .csv files for switching to AgS

### ISSUE THREAD

**Issue by mr-gosh** [agstrike/silverstrike#83]
For People who have used moneyplex, moneypenny quicken etc. there should be an import of .csv transaction lists - that would spread the user base a lot.

**Comment by simhnna** [comment]
I want to have a generic import method but that's currently a lot of work. I'm working on a new frontend that should make implementing such things easier.

Can you share the general structure of the csv exports of what you would want to transfer from?

**Comment by mr-gosh** [comment]
I have used moneyplex for several years now - but the development is stalled in my eyes - so i want to migrate the database to something like silverstrike - but without the loss of transaction data - so i could create some dumps and anonymize it. than i could upload them here for futher review...?

**Comment by simhnna** [comment]
Yeah just upload a file with some sample data, then I can write an importer.
